webapp/src/main/java/com/box/l10n/mojito/quartz/multi/QuartzMultiSchedulerConfigurationProperties.java

```diff
@@ -0,0 +1,74 @@
+package com.box.l10n.mojito.quartz.multi;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties("l10n.org.multi-quartz")
+@ConditionalOnProperty(name = "l10n.org.multi-quartz.enabled", havingValue = "true")
+public class QuartzMultiSchedulerConfigurationProperties {
+
+  Map<String, Map<String, String>> schedulers;
+
+  List<SchedulerConfigurationProperties> schedulerConfigurationProperties = new ArrayList<>();
+
+  public Map<String, Map<String, String>> getSchedulers() {
+    return schedulers;
+  }
+
+  public void setSchedulers(Map<String, Map<String, String>> schedulers) {
+    this.schedulers = schedulers;
+  }
+
+  public List<SchedulerConfigurationProperties> getSchedulerConfigurationProperties() {
+    return schedulerConfigurationProperties;
+  }
+
+  @PostConstruct
+  public void init() {
+    schedulerConfigurationProperties.addAll(extractSchedulerConfigs());
+  }
+
+  private List<SchedulerConfigurationProperties> extractSchedulerConfigs() {
+    List<SchedulerConfigurationProperties> schedulerProperties = new ArrayList<>();
+
+    for (Map.Entry<String, Map<String, String>> entry : schedulers.entrySet()) {
+      SchedulerConfigurationProperties config = new SchedulerConfigurationProperties();
+      config.setName(entry.getKey());
+      for (Map.Entry<String, String> quartzEntry : entry.getValue().entrySet()) {
+        config.getQuartz().put(quartzEntry.getKey().replace("quartz.", ""), quartzEntry.getValue());
+      }
+      schedulerProperties.add(config);
+    }
+    return schedulerProperties;
+  }
+}
+
+class SchedulerConfigurationProperties {
+
+  private String name;
+
+  private Map<String, String> quartz = new HashMap<>();
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public Map<String, String> getQuartz() {
+    return quartz;
+  }
+
+  public void setQuartz(Map<String, String> quartz) {
+    this.quartz = quartz;
+  }
+}
```

Comment:
A UT in this commit, just for this class would be handy (commented in the other commit regarding test having dependency on other class)